Common/DataModel/EseTable.h

```diff
@@ -0,0 +1,45 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+// q vector framework with ESE (20/08/2024)
+//
+/// \author Joachim Hansen <joachim.hansen@cern.ch>
+//
+
+#ifndef COMMON_DATAMODEL_ESETABLE_H_
+#define COMMON_DATAMODEL_ESETABLE_H_
+
+#include <vector>
+
+#include "Framework/AnalysisTask.h"
+#include "Framework/ASoAHelpers.h"
+
+namespace o2::aod
+{
+namespace qVector
+{
+DECLARE_SOA_COLUMN(QFV0ARe, qFV0ARe, float);
+DECLARE_SOA_COLUMN(QFV0AIm, qFV0AIm, float);
+DECLARE_SOA_COLUMN(QFT0CRe, qFT0CRe, std::vector<float>);
+DECLARE_SOA_COLUMN(QFT0CIm, qFT0CIm, std::vector<float>);
+DECLARE_SOA_COLUMN(QPERCFT0C, qPERCFT0C, std::vector<float>);
+DECLARE_SOA_COLUMN(QTEST, qTEST, float);
+} // namespace qVector
```

Comment:
```suggestion
} // namespace q_vector
```

Reply:
Hi Vit, using the PR to commit suggestions is very smart. But my lack of git knowledge is giving me some quite big git issues now where I cannot push to my repository, since there's differences in files that are now modified several places. Would it be better to close this PR and make a new one?

Reply:
I force pushed changes now, but I should have added your suggestions..

Reply:
Hi @joachimckh , please try to avoid overwriting the PR history with force-pushing. Details are explained in the guidelines for contributors: https://aliceo2group.github.io/analysis-framework/docs/gettingstarted/contributingtocode.html#pull-request-review
If commits are added to your branch on GitHub, you should sync your local branch with it using `git pull --rebase origin <branch>`, where in this case your branch is `master`. This will take the commit history from your remote branch and put your additional local commits on top of them so that you can cleanly push them to your remote branch. Again, more details are on the documentation pages I linked.

Reply:
@vkucera  Alright, thanks for all the information. I will avoid it in the future. 

The changes I put, is the renaming of the variable that you requested, as well as a small change in FFitWeights.